src/modules/mavlink/mavlink_main.cpp

```diff
@@ -543,7 +543,7 @@ Mavlink::forward_message(const mavlink_message_t *msg, Mavlink *self)
 			const mavlink_msg_entry_t *meta = mavlink_get_msg_entry(msg->msgid);
 
 			int target_system_id = 0;
-			int target_component_id = 233;
```

Comment:
What is/was 233?

Reply:
Nobody knows, not even @sanderux who put it there initially.

Reply:
I seem to remember this was some "all components" magic number but i cannot find any reference to it anymore.

Reply:
"All components" is 0 - https://mavlink.io/en/messages/common.html#MAV_COMP_ID_ALL

This is empty and unreserved. If it needs to be all components then it should be changed to zero. If it means something else then it will need a new component ID. As it is, we're likely to eventually overwrite this, and you will end up with unexpected behaviour.

Reply:
Yes i realize that, i just can't imagine i made one up, nor can i imagine lorenz would have approved it if i did.